doc/source/how-to/code/docs.yml

```diff
@@ -26,7 +26,21 @@ doc-deploy-stable:
   needs: doc-deploy-dev
   steps:
     - name: "Deploy stable documentation"
-      uses: pyansys/actions/doc-deploy-stable@v3
+      uses: ansys/actions/doc-deploy-stable@v4
       with:
         cname: ${{ env.CNAME }}
         token: ${{ secrets.GITHUB_TOKEN }}
+
+doc-deploy-index:
+  name: "Index the documentation and scrap using pymeilisearch"
+  runs-on: ubuntu-latest
+  needs: doc-deploy
+  if: github.event_name == 'push'
+  steps:
+    - name: "Deploy the latest documentation index"
+      uses: ansys/actions/doc-deploy-index@v4.1
+      with:
+          cname: "<library>.docs.pyansys.com/version/<version-number>"
```

Comment:
Handling yet another location for the version number might be "undesired"... Is there any way we can automate this? I'd avoid having it in more than one location that version number...